src/Controls/samples/Controls.Sample/ViewModels/CoreViewModel.cs

```diff
@@ -15,6 +15,9 @@ public class CoreViewModel : BaseGalleryViewModel
 			new SectionModel(typeof(AppThemeBindingPage), "AppThemeBindings",
 				"Devices typically include light and dark themes, which each refer to a broad set of appearance preferences that can be set at the operating system level. Applications should respect these system themes, and respond immediately when the system theme changes."),
 
+			new SectionModel(typeof(BordersPage), "Borders",
+				"Draws a border around a Layout."),
```

Comment:
this should be around any control right?

Reply:
```suggestion
				"Draws a border around a View, Layout, or Control."),
```